autosklearn/experimental/balanced_accuracy/askl2_portfolios/RF_SH-eta4-i_holdout_iterative_es_if.json

```diff
@@ -1,1197 +1 @@
.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.1953125": 0, "0.390625": 1, "0.78125": 2, "1.5625": 3, "3.125": 4, "6.25": 5, "12.5": 6, "25": 7, "50": 8, "100": 9}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.1953125": 0, "0.390625": 1, "0.78125": 2, "1.5625": 3, "3.125": 4, "6.25": 5, "12.5": 6, "25": 7, "50": 8, "100": 9}, {"0.1953125": 0, "0.390625": 1, "0.78125": 2, "1.5625": 3, "3.125": 4, "6.25": 5, "12.5": 6, "25": 7, "50": 8, "100": 9}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.1953125": 0, "0.390625": 1, "0.78125": 2, "1.5625": 3, "3.125": 4, "6.25": 5, "12.5": 6, "25": 7, "50": 8, "100": 9}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}, {"0.390625": 0, "0.78125": 1, "1.5625": 2, "3.125": 3, "6.25": 4, "12.5": 5, "25": 6, "50": 7, "100": 8}], "config_to_task": {"b8184c0": [340], "05327ff": [2119], "73fc45b": [2122], "d38f297": [2123], "d033742": [3047], "8c75f1c": [3049], "074da43": [75089], "a479a3d": [75098], "d54bc21": [75148], "8cd6304": [75157], "b86e0b2": [75159], "978b964": [75171], "e249036": [126030], "f187687": [146574], "2f6c8d6": [146578], "a3ba205": [146597], "d618703": [146602], "c437bea": [166866], "77764f9": [166957], "05690ed": [167085], "c2b918a": [167088], "eb6f7f1": [167089], "c29f8bf": [167099], "2d4a38e": [167101], "7cab112": [189779], "68e6606": [189836], "811cba1": [189843], "fe0e803": [189863], "c832fa9": [189881], "a2734c9": [189882], "564103f": [189884], "07bebab": [189893]}, "input_directory": "/work/dlclarge2/feurerm-askl2paper/askl2paper/2020_IEEE_Autosklearn_experiments/experiment_scripts/60MIN/ASKL_getportfolio/RF_SH-eta4-i_holdout_iterative_es_if", "fidelities": "SH", "portfolio_size": 32, "seed": 0, "max_runtime": null, "start_time": 1622279454.1839142, "end_time": 1622279506.850947, "wallclock_time": 52.66703271865845}
```

Comment:
Could you please keep the formatting intact (should be achieved with indent = 4 when writing the json)?

Reply:
While this is a topic, should we sort keys as well where possible?
```python
with open(...) as file:
    json.dump(dictionary, file, indent=4, sort_keys=True)
```

Reply:
We would have to do so in a different repo.